serving/src/main/java/ai/djl/serving/http/InferenceRequestHandler.java

```diff
@@ -131,14 +133,23 @@ protected void handleRequest(
                                 });
                 break;
             case "invocations":
-                handleInvocations(ctx, req, decoder, null);
+                if (Boolean.parseBoolean(Utils.getenv("MESSAGES_API_ENABLED"))) {
+                    handleChatCompletions(ctx, req, decoder, null);
```

Comment:
Can we move these changes to a plugin? This seems like another prediction format for compatibility, so it is better if we keep it external from our core design to keep DJL Serving clean